.github/workflows/check-nix-format.yml

```diff
@@ -46,6 +46,7 @@ jobs:
           NIX_FMT_PATHS_VSCODE_EXTS: pkgs/applications/editors/vscode/extensions
           NIX_FMT_PATHS_PHP_PACKAGES: pkgs/development/php-packages
           NIX_FMT_PATHS_BUILD_SUPPORT_PHP: pkgs/build-support/php
+          NIX_FMT_PATHS_PYTHON_PACKAGES: pkgs/development/python-modules
```

Comment:
This is a little bit problematic for directories that frequently get PRs with new files, because those files are not formatted properly yet and would break master when merged

Reply:
I agree that the overall situation is unfortunate, but maybe this is less bad than having people reformat every file individually as they touch it?

Reply:
What I understand from this is:
- The CI action does not check new files in a PR
- The CI action will notice wrong formatting once merged into master, and subsequentl fail

Is that anything that can be sorted out in a reasonable timeframe?

Reply:
I'd be fine with deferring the CI action for now, but going for the treewide reformat.